tests/pauli/test_conversion.py

```diff
@@ -266,12 +363,6 @@ def test_operator(self, op, ps):
         """Test that PL arithmetic op is properly cast to a PauliSentence."""
         assert pauli_sentence(op) == ps
 
-    @pytest.mark.parametrize("op, ps", operator_ps)
-    def test_operator_private_ps(self, op, ps):
-        """Test that a correct pauli sentence is computed when passing an arithmetic operator and not
-        relying on the saved op._pauli_rep attribute."""
-        assert qml.pauli.conversion._pauli_sentence(op) == ps  # pylint: disable=protected-access
```

Comment:
Why was this removed ?

Reply:
I don't remember deleting these. It must be a mistake, let me restore them.